app/src/main/java/it/kapfer/digitalesamt/liberator/ModuleMain.kt

```diff
@@ -1,54 +1,90 @@
 package it.kapfer.digitalesamt.liberator
 
 import android.content.res.XModuleResources
+import android.os.Build
 import de.robv.android.xposed.IXposedHookLoadPackage
 import de.robv.android.xposed.IXposedHookZygoteInit
 import de.robv.android.xposed.XC_MethodReplacement
 import de.robv.android.xposed.XposedBridge
 import de.robv.android.xposed.XposedHelpers
 import de.robv.android.xposed.callbacks.XC_LoadPackage
+import java.io.File
 
 // Classes to hook in Digitales Amt app
 const val DEVICE_INTEGRITY_CHECK_CLASS: String = "at.asitplus.utils.deviceintegrity.DeviceIntegrityCheck"
 // Classes to hook in FON [+] app
-const val ROOTBEER_CLASS: String = "com.scottyab.rootbeer.b"
+const val ROOTBEER_CLASS: String = "com.scottyab.rootbeer.RootBeer"
```

Comment:
Shouldn't there be two types of rootbeer hooks as well, just as for the attestation helper?

Reply:
Yeah, forgot about that one. Will patch later

Reply:
I couldn't test, as I don't have the old apk